codex-rs/cloud-tasks/src/bin/conncheck.rs

```diff
@@ -0,0 +1,91 @@
+use codex_cloud_tasks::util::build_chatgpt_headers;
+use codex_cloud_tasks::util::normalize_base_url;
+use codex_cloud_tasks::util::set_user_agent_suffix;
+use codex_cloud_tasks_client::CloudBackend;
+use codex_cloud_tasks_client::HttpClient;
+use codex_cloud_tasks_client::MockClient;
+use reqwest::header::AUTHORIZATION;
+use reqwest::header::HeaderName;
+use reqwest::header::USER_AGENT;
+use std::sync::Arc;
+use std::time::Duration;
+use tokio::time::timeout;
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
```

Comment:
Are these vibecoded bins you used to test while building? Do we need to commit these?